src/server/rest/v1/service/AuthorizationService.ts

```diff
@@ -57,40 +57,24 @@ export default class AuthorizationService {
   }
 
   public static async checkAndGetSiteAuthorizationFilters(tenant: Tenant, userToken: UserToken,
-      filteredRequest: HttpSiteRequest): Promise<AuthorizationFilter> {
+      filteredRequest: Partial<Site>, authAction: Action): Promise<AuthorizationFilter> {
```

Comment:
Are you sure about that one? It should be `Partial<HttpSiteRequest>`.

Reply:
You're right.